docker-compose.yml

```diff
@@ -18,6 +18,39 @@ services:
       - phabricator.test
       - bmo.test
 
+  lando-ui.test:
+    image: mozilla/landoui
```

Comment:
Ok. So I can create PRs to fix the tasks.py file in both lando-ui and lando-api to be more consistent with the dockerhub repos.